### ISSUE TITLE
Support for chromatic fonts

### ISSUE THREAD

**Issue by mateusfccp** [westracer/fontify#14]
Flutter supports rendering chromatic icons.

However, when I use `fontify` on a set of `.svg` files in which some of them have colors, the resulting `.otf` has a single and monochromatic shape.

For example, for this SVG:

<img width="49" alt="Screen Shot 2021-05-06 at 15 58 12" src="https://user-images.githubusercontent.com/4605213/117351254-e02e1f80-ae83-11eb-94bd-29789b846fd7.png">

The result glyph is:
<img width="39" alt="Screen Shot 2021-05-06 at 15 58 43" src="https://user-images.githubusercontent.com/4605213/117351300-f0de9580-ae83-11eb-9723-4ff51694e970.png">

Is there a way to preserve SVG colors when generating an `.otf` with `fontify`?

**Comment by mateusfccp** [comment]
As I was giving a look, it seems `fontify` does not works with SVG table, and I think this is the reason my SVGs colors disappear. Is that right?

**Comment by mateusfccp** [comment]
Ok, so I made some tests and it seems that even if I put an SVG Table in the font generated by `fontify` with the desired SVG, Flutter will render it like described in the issue, which makes me think that Flutter doesn't support SVG tables at all.

(I patched SVGs into the generated font with [`opentype-svg`](https://github.com/adobe-type-tools/opentype-svg))

I also tested COLR/CPAL fonts and they also didn't work properly (although they at least kept the shape as expected, but using the `IconTheme` color).

The only font that I tested and properly displayed colored glyphs was Noto Sans Emoji, which is, apparently, a CBDT font. This is disappointing, as bitmaps doesn't scale well...

Anyway, if we want to proceed with this idea, we will have to process the given SVG into a CBDT font.
